cfme/infrastructure/virtual_machines.py

```diff
@@ -1307,3 +1313,15 @@ class VmEngineRelationship(CFMENavigateStep):
     def step(self):
         self.prerequisite_view.toolbar.configuration.item_select(
             'Edit Management Engine Relationship')
+
+
+@navigator.register(Vm, 'Explore')
+class VmExplore(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
+    VIEW = InfraVmDetailsView
+
+    def step(self, *args, **kwargs):
+        self.prerequisite_view.navigation.select('Compute', 'Infrastructure', '/vm_infra/explorer')
+
+    def am_i_here(self, *args, **kwargs):
+        return match_page(summary='All VMs & Templates')
```

Comment:
REQUIRED: match_page is depreacated. Please use some_view.is_displayed instead. It will be called by default implementation of am_i_here